docs/changelog.md

```diff
@@ -59,21 +59,37 @@ For more information on features and bug fixes in 0.6.0, see the GitHub mileston
 
 #### Upgrade Instructions
 
-##### Serialization JanusGraphPredicates has changed
+##### Serialization of JanusGraph predicates has changed
 
-GraphSON and Gryo Serialization has changed so both version need to be aligned.
+The serialization of JanusGraph predicates has changed in this version for both 
+GraphSON and Gryo. It is therefore necessary to update both the client and 
+the server to this version in parallel as the server will not be able to 
+deserialize a JanusGraph predicate that was serialized by a client prior 
+to version 0.6.0 once it was updated to version 0.6.0.
 
-##### GraphBinary is now support
+##### GraphBinary is now supported
 
-If you want to use GraphBinary, you have to add following to the `gremlin-server.yaml` after the keyword `serializers`. 
-This will added the support on the server site. For client site, you have to checkout based on the driver.
+[GraphBinary](http://tinkerpop.apache.org/docs/current/dev/io/#graphbinary) is a 
+new binary serialization format from TinkerPop that supersedes Gryo and it will 
+eventually also replace GraphSON. GraphBinary is language independent and has a 
+low serialization overhead which results in an improved performance.
+
+If you want to use GraphBinary, you have to add following to the `gremlin-server.yaml` 
+after the keyword `serializers`. This will add the support on the server site. 
 
 ```yaml
     - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1, 
         config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
     - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1, 
         config: { serializeResultToString: true }}
 ```
+JanusGraph Extensions for GraphBinary are currently supported by following clients:
+
+| Language | Version |
```

Comment:
Not sure if this table adds much value. Just saying that this is currently only supported in our Java driver would be enough in my opinion (in addition with a short explanation / link how that can be configured). We don't have a Python driver yet, so I wouldn't even mention that here.